docs/release/trg-0/TRG 6.02 API Versioning.md

```diff
@@ -0,0 +1,62 @@
+---
+title: TRG 6.02 - API Versioning
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: "mandatory after": 01st of May 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 30-Nov-2023  | Initial contribution                   |
+
+## Why
+
+For publicly available APIs a versioning mechanism is required, so that all APIs are versioned the same way.
+
+Note that for APIs accessible only via EDC, the API-version is included in the EDC asset and not the URL. This is described in CX-0018.
+
+## Description
+
+For public REST-APIs (= not hidden behind an EDC aspect), the Version number **must** be included in the URL.

```

Comment:
@SebastianBezold I have updated the formatting as requested (bold instead of upper case)